src/main/java/org/aksw/simba/lemming/metrics/single/edgemanipulation/VertexDegrees.java

```diff
@@ -0,0 +1,181 @@
+package org.aksw.simba.lemming.metrics.single.edgemanipulation;
+
+import org.aksw.simba.lemming.ColouredGraph;
+
+import grph.Grph.DIRECTION;
+import it.unimi.dsi.fastutil.ints.IntIterator;
+import it.unimi.dsi.fastutil.ints.IntOpenHashSet;
+import it.unimi.dsi.fastutil.ints.IntSet;
+
+/**
+ * Class for storing in and out degree of vertices.
+ * 
+ * @author Atul
+ *
+ */
+public class VertexDegrees {
+
+	// Maps for storing in-degree and out-degree for different vertices
+	private int mMapVerticesinDegree[];
```

Comment:
Please make sure that you use camelCase writing. In several attribute and method names, not all words within the name start with a capital letter. In this line for example, the `in` should be `In`.

Reply:
Done. I have corrected the variable and method names.